charger/openwb.go

```diff
@@ -185,7 +185,11 @@ func (m *OpenWB) Status() (api.ChargeStatus, error) {
 	if err != nil {
 		return api.StatusNone, err
 	}
-	return api.ChargeStatusString(status)
+	cs, err2 := api.ChargeStatusString(status)
```

Comment:
Passt- gehört aber in den `Enabled` call. Aufwand entsteht nicht da wir die Daten dank MQTT eh haben.

Reply:
Geschmackssache aber habs geändert